bin/node/src/chain_spec/cli.rs

```diff
@@ -36,28 +24,29 @@ pub struct ChainParams {
     #[arg(long, default_value = "DZERO")]
     token_symbol: String,
 
-    /// AccountIds of authorities forming the committee at the genesis (comma delimited)
+    /// all account ids that needs to have session keys generated when bootstraping chain (comma delimeted)
     #[arg(long, value_delimiter = ',', value_parser = parse_account_id, num_args = 1..)]
     account_ids: Vec<AccountId>,
 
+    /// AccountIds of authorities forming the committee at the genesis (comma delimited)
+    /// If empty, then `--account-ids` are used as authorities
```

Comment:
we should probably add a note that it should be a subset of `account_ids`

Reply:
Yes, and let's throw an error if it is not